### PR TITLE
Displayed field representation of state_hash

### PR DESCRIPTION
**Pull request by wu-s-john**
Killed some duplication in interfaces in state_hash

Graphql Query:
```
query MyQuery {
  blocks(first: 2) {
    nodes {
      stateHash
      protocolStateProof {
        a
        b
      }
    }
  }
}

```

Output:
```
{
  "data": {
    "blocks": {
      "nodes": [
        {
          "stateHash": "TWogr3GykfwAkjiqur9vNJsokVCurdeQGjNbrJNY5owZK1w1a3TRnzuxKpy5tSi2",
          "protocolStateProof": {
            "a": [
              "218659069152196389374111529990691173023664859457440667326700485601809050062039006656366448",
              "70886220239726735381157050385259244767473536827419312615561230137655941915903779944887857"
            ],
            "b": [
              [
                "234794458455109139771021028430373306648733249042844481679643770635195227346453494440140199",
                "310459695560964773513966763739173172449182746511934905624017695432103792528600669690831520",
                "287707961161242561272416368712477704905999404761858058654494123333291078409490993969841666"
              ],
              [
                "53000926026643226274315269714342040191885030637284293362177363710887191112902109058217423",
                "33367879529490530920402612096772926020222968203684790091504340518569067980334948331781466",
                "67250666468171348745653280035482192547991887323399715708184941898959275650427517469203957"
              ]
            ]
          }
        },
        {
          "stateHash": "TWogAexosj5k39YXMLYG5Dc12xAbGDDgzh2p7jmV4AmNdytfRAv3xG2wks24qJda",
          "protocolStateProof": {
            "a": [
              "218659069152196389374111529990691173023664859457440667326700485601809050062039006656366448",
              "70886220239726735381157050385259244767473536827419312615561230137655941915903779944887857"
            ],
            "b": [
              [
                "234794458455109139771021028430373306648733249042844481679643770635195227346453494440140199",
                "310459695560964773513966763739173172449182746511934905624017695432103792528600669690831520",
                "287707961161242561272416368712477704905999404761858058654494123333291078409490993969841666"
              ],
              [
                "53000926026643226274315269714342040191885030637284293362177363710887191112902109058217423",
                "33367879529490530920402612096772926020222968203684790091504340518569067980334948331781466",
                "67250666468171348745653280035482192547991887323399715708184941898959275650427517469203957"
              ]
            ]
          }
        }
      ]
    }
  }
}

```

Checklist:

- [ ] Tests were added for the new behavior
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them:

Closes #0000
Closes #0000

### CODE REVIEW

File: src/lib/coda_graphql/coda_graphql.ml [hunk 1]
```diff
@@ -726,6 +726,11 @@ module Types = struct
             ~args:Arg.[]
             ~resolve:(fun _ {With_hash.hash; _} ->
               Stringable.State_hash.to_base58_check hash )
+        ; field "stateHashField" ~typ:(non_null string)
+            ~doc:"Bigint field-element representation of stateHash"
```
Comment (cmr) [inline src/lib/coda_graphql/coda_graphql.ml]: According to the example in the PR description this should be base58check. Did you change your mind? base58 is the most useful representation here right?

Comment (bkase) [reply src/lib/coda_graphql/coda_graphql.ml]: We decided to keep both, base58check is useful as that’s what we use within the daemon and display, but the raw number exposed means we don’t need a base58 decoder in JavaScript right now

Comment (bkase) [reply src/lib/coda_graphql/coda_graphql.ml]: We still have the other stateHash, right John?

Comment (wu-s-john) [reply src/lib/coda_graphql/coda_graphql.ml]: yes

Comment (Schmavery) [reply src/lib/coda_graphql/coda_graphql.ml]: hmm https://www.npmjs.com/package/base58check

**Pull request merged**